camp/jbolotski/beam_break_motor/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.commands.RunMotorFromBeamBreak;
+import frc.robot.sim.PhysicsSim;
+import frc.robot.subsystems.BeamBreakMotor;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+  // The robot's subsystems are defined here...
+  private final BeamBreakMotor m_beamBreakMotor = new BeamBreakMotor(Constants.beamBreakMotorSpeed);
+  private final ManualInputInterfaces m_manualInputs = new ManualInputInterfaces();
+
+  // The robot's commands are defined here...
+  private final RunMotorFromBeamBreak bbAutoCommand = new RunMotorFromBeamBreak(m_beamBreakMotor, m_manualInputs);
+
+  /** The container for the robot. Contains subsystems, OI devices, and commands. */
+  public RobotContainer() {
+    // Configure the button bindings
+    configureButtonBindings();
+  }
+
+  /**
+   * Use this method to define your button->command mappings. Buttons can be created by
+   * instantiating a {@link GenericHID} or one of its subclasses ({@link
+   * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
+   * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
+   */
+  private void configureButtonBindings() {}
```

Comment:
So if we wanted to go the Trigger route as described above, the scheduling goo would go in this method.

Reply:
done.